receiver/awscontainerinsightreceiver/internal/k8swindows/extractors/extractorhelpers.go

```diff
@@ -1,9 +1,47 @@
 package extractors
 
 import (
+	"fmt"
+
 	stats "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
 )
 
+// convertCPUStats Convert kubelet CPU stats to Raw CPU stats
+func convertCPUStats(kubeletCPUStat *stats.CPUStats) *CPUStat {
+	var cpuStat CPUStat
+	if kubeletCPUStat.UsageCoreNanoSeconds != nil {
+		cpuStat.UsageCoreNanoSeconds = *kubeletCPUStat.UsageCoreNanoSeconds
+	}
+	if kubeletCPUStat.UsageNanoCores != nil {
+		cpuStat.UsageNanoCores = *kubeletCPUStat.UsageNanoCores
+	}
+	return &cpuStat
```

Comment:
Why do the helpers need to return a pointer?
I wonder why they even need to take in pointers

I realize the software around it expects pointers but maybe we should do the `&` reference at higher scope

Reply:
>Why do the helpers need to return a pointer?
yes chad i think you are right. It makes more sense to not make these as pointers since they will never be nil and will have have default values for each field in RawMetric. I changed this and commit the changes.

Reply:
Hi chad, I thought about it.
Even though we can avoid using pointers here but I see benefit of using pointers.
These structs are supposed to be large in the sense that they will have many fields, if we start passing these structs by value, it's will be very heavy in processing since go will have to copy whole struct when passing them as arguments in func calls. It is better to have them pass by reference since it will go processing.

what do you think? if you still think pointers are unnecessary, then i remove pointers.

Reply:
This is not a hot path.  This code will get called once every 15 seconds or every 60 seconds or whatever.  Go is designed to pass by value.  Passing by reference is an exception IMO.  These structs are not big, maybe 100 bytes

Reply:
Sounds good, I have removed pointers references.